### PR TITLE
Add skeleton subset service

### PR DESCRIPTION
**Pull request by jonescc**

### CODE REVIEW

File: grails-app/conf/BuildConfig.groovy [hunk 1]
```diff
@@ -67,10 +68,6 @@ grails.project.dependency.resolution = {
         runtime ":database-migration:1.4.0"
         runtime ":jquery:1.11.1"
 
-        // Uncomment these to enable additional asset-pipeline capabilities
-        //compile ":sass-asset-pipeline:1.9.0"
-        //compile ":less-asset-pipeline:1.10.0"
-        //compile ":coffee-asset-pipeline:1.8.0"
-        //compile ":handlebars-asset-pipeline:1.3.0.3"
+        build ":tomcat:$grailsVersion"
```
Comment (danfruehauf) [inline grails-app/conf/BuildConfig.groovy]: I think for portal I had to use here something like 7.0.xx. Seems like 2.4.4 might be too old (or non-existent).

Are you running it with `grails run-app`?

File: grails-app/controllers/au/org/emii/netcdfsubset/SubsetController.groovy [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import org.joda.time.*
+import org.joda.time.format.*
+import org.springframework.validation.Errors
+
+class SubsetController {
+    def subsetService
+    
+    def messageSource
+
+    def index (SubsetRequest subsetRequest) {
```
Comment (danfruehauf) [inline grails-app/controllers/au/org/emii/netcdfsubset/SubsetController.groovy]: Redundant space after `index`

**Comment by danfruehauf** [comment]
@jonescc That's good, just the couple of minor issues I mentioned.

File: grails-app/conf/BuildConfig.groovy (hunk 1 above)
Comment (jonescc) [inline grails-app/conf/BuildConfig.groovy]: I copied this from go-go-duck!! without looking at what it did.   I'll check

Comment (danfruehauf) [inline grails-app/conf/BuildConfig.groovy]: GGD is running 2.2.0. Porting portal to 2.4.4 I had to put 7.0.xx there...

Comment (jonescc) [inline grails-app/conf/BuildConfig.groovy]: Yep.  Needs to be updated.  Is go-go-duck runing on 2.4.4?

Comment (jonescc) [inline grails-app/conf/BuildConfig.groovy]: Applies to 2.3 and up.  My bad

**Comment by jonescc** [comment]
@danfruehauf comments addressed

**Pull request merged**